packages/web/config/feature-flag.ts

```diff
@@ -81,6 +81,13 @@ export const ExternalIncentiveGaugeAllowList: {
       denom:
         "ibc/A0CC0CF735BFB30E730C70019D4218A1244FF383503FF7579C9201AB93CA9293",
     },
+    ],
+  "788": [
+    {
+      gaugeId: "29476",
```

Comment:
I think you made the gauge correctly, and that this gauge ID is just a typo. Looks like it should be 29872 based on the tx you gave to support. If you could just correct the gauge ID and move this pool object in chronological order (so, after pool 787), then this should be good to merge.